### DISCUSSION TITLE
Welcome to StarRocks Discussions!

### DISCUSSION THREAD

**Discussion by imay** [StarRocks/starrocks#2349] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eden0907** [comment]
Hello,
is there a documentation on how to do a simple update on data
like : update table1 set siteid = 2 where siteid = 1;   
and how to update lot of records based on an another table data,
like (update table1 dest set (siteid, username) = (select siteid, nickname from an_another_table src where dest.siteid = src.siteid ...)
thank you,

**Reply by alberttwong** [reply]
You should be able to do those operations now.
